Make the test pass, having no clue what am I changing at all.

Lubomir Rintel <lkundrak@v3.sk>

diff -up libsamplerate-0.1.7/tests/callback_test.c.test libsamplerate-0.1.7/tests/callback_test.c
--- libsamplerate-0.1.7/tests/callback_test.c.test	2009-05-08 11:21:42.000000000 +0200
+++ libsamplerate-0.1.7/tests/callback_test.c	2009-05-08 11:21:09.000000000 +0200
@@ -137,7 +137,7 @@ callback_test (int converter, double src
 
 	src_state = src_delete (src_state) ;
 
-	if (fabs (read_total - src_ratio * ARRAY_LEN (test_callback_data.data)) > src_ratio)
+	if (fabs (read_total - floor (0.5 + src_ratio * ARRAY_LEN (test_callback_data.data))) > ceil (src_ratio))
 	{	printf ("\n\nLine %d : input / output length mismatch.\n\n", __LINE__) ;
 		printf ("    input len  : %d\n", ARRAY_LEN (test_callback_data.data)) ;
 		printf ("    output len : %ld (should be %g +/- %g)\n\n", read_total,
